packages/spinners/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@truffle/spinners",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.1.0-0",
```

New packages should start at v0.1.0, and we put the `-0` there for the time being, until the first publish (when we increment to 0.1.0)